tap_pendo/streams.py

```diff
@@ -29,6 +29,15 @@
 # timeout request after 300 seconds
 REQUEST_TIMEOUT = 300
 
+FACTOR = 2
+COUNT = 5
```

Comment:
Can you please remove this variable as it is not used anywhere in the code?

Reply:
Removed unused variable.